tictactoe.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Play Tic Tac Toe</title>
+	<link rel="stylesheet" type="text/css" href="tictactoe.css">
+</head>
+<body>
+<h1>Play Tic Tac Toe</h1>
+<div id="grid">
+	<div class= "row" id="row1">
+		<div class="tile" id="tile1"></div>
+		<div class="tile" id="tile2"></div>
+		<div class="tile" id="tile3"></div>
+	</div>
+	<div class= "row" id="row2">
+		<div class="tile" id="tile4"></div>
+		<div class="tile" id="tile5"></div>
+		<div class="tile" id="tile6"></div>
+	</div>
+	<div class= "row" id="row3">
+		<div class="tile" id="tile7"></div>
+		<div class="tile" id="tile8"></div>
+		<div class="tile" id="tile9"></div>
+	</div>	
+</div>
+<div id="button">
+<button type="button" id="reset">Reset</button>
+</div>
+
+<script type="text/javascript" src="tictactoe.js"></script>
+</body>
+</html>
```

Comment:
Nice page layout, especially when it comes to laying out divs in a row/column format. We'll see that pattern a lot when working with CSS frameworks.